Dockerfile.test

```diff
@@ -1,19 +1,21 @@
-FROM alpine:3.3
+FROM ubuntu:14.04
```

Comment:
I'll test with 16.04. I had to move away from alpine due to segfaults in its libc (musl), arising from possibly inefficient  C + python wrapping in psvmi. I'll flag this as in 'issue'